components/kyma-environment-broker/internal/provisioner/graphqlizer.go

```diff
@@ -55,6 +55,9 @@ func (g *Graphqlizer) ClusterConfigToGraphQL(in gqlschema.ClusterConfigInput) (s
 		{{- if .GardenerConfig }}
 		gardenerConfig: {{ GardenerConfigInputToGraphQL .GardenerConfig }},
 		{{- end }}
+		{{- if .Administrators }}
+		administrators: {{.Administrators | marshal }},
+		{{- end }}
 	}`)
```

Comment:
what about `UpgradeRuntimeInputToGraphQL` method? In general how upgrade should behave?

Reply:
We have to more discuss about this, so it will be done in next PR.